python/python/glide/async_commands/server_modules/json.py

```diff
@@ -139,6 +139,55 @@ async def get(
     return cast(bytes, await client.custom_command(args))
 
 
+async def arrappend(
+    client: TGlideClient,
+    key: TEncodable,
+    values: List[TEncodable],
+    path: Optional[TEncodable] = None,
+) -> TJsonResponse[int]:
+    """Appends one or more `values` to the JSON array at the specified `path` within the JSON document stored at `key`.
+
+    See https://valkey.io/commands/json.append/ for more details.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        key (TEncodable): The key of the JSON document.
+        values (TEncodable): The values to append to the JSON array at the specified path.
+        path (Optional[TEncodable]): Represents the path within the JSON document where the `values` will be appended.
+            Defaults to None.
+
+    Returns:
+        TJsonResponse[int]:
+            For JSONPath (`path` starts with `$`):
+                Returns a list of integer replies for every possible path, indicating the new length of the new array after appending `values`,
+                or None for JSON values matching the path that are not an array.
+                If `key` doesn't exist, an error is raised.
+            For legacy path (`path` doesn't start with `$`):
+                Returns the length of the new array after appending `values` to the array at `path`.
+                If multiple paths match, the length of the last updated array is returned.
```

Comment:
need to check this, @Yury-Fridlyand maybe you can understand this?

```
127.0.0.1:6379> JSON.SET k1 $ '{"a": 1, "b": ["one", "two"], "c": {"b": ["three"]}}'
OK
127.0.0.1:6379> JSON.ARRAPPEND k1 *.b '"five"'
(integer) 2
127.0.0.1:6379> json.get k1 $
"[{\"a\":1,\"b\":[\"one\",\"two\"],\"c\":{\"b\":[\"three\",\"five\"]}}]"`
```
i expected it to change the first b as well

https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/json-arrappend.html

Reply:
`*.b* means matches the path "c.b".  the first "b" doesn't have the "." in the path.

Maybe you meant to use something like `$[b]`?

Reply:
Hassan have answered on slack